alexandria/core/apps.py

```diff
@@ -11,10 +11,15 @@ def ready(self):
         # to avoid recursive import error, load extension classes
         # only once the app is ready
         from .models import VisibilityMixin, PermissionMixin
+        from .serializers import BaseSerializer
 
         PermissionMixin.permission_classes = [
             import_string(cls) for cls in settings.PERMISSION_CLASSES
         ]
         VisibilityMixin.visibility_classes = [
             import_string(cls) for cls in settings.VISIBILITY_CLASSES
         ]
+
+        BaseSerializer.validation_classes = [
+            import_string(cls) for cls in settings.VALIDATION_CLASSES
+        ]
```

Comment:
Do we still potentially allow multiple classes? Would be consistent to the other extensions...

Reply:
Yes, we support multiple classes. What we don't allow is using the same validator method for multiple models